docs/dev/README.md

```diff
@@ -1,35 +1,128 @@
 ---
-title: Introduction to Agent-based Integrations
+title: Build an Integration
 kind: documentation
+description: Learn how to develop and publish an offering on the Integrations page.
 aliases:
-  - /guides/agent_checks/
-  - /agent/agent_checks
-  - /developers/agent_checks/
+- /guides/agent_checks/
+- /agent/agent_checks
+- /developers/agent_checks/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/README.md"
+further_reading:
```

Comment:
Can we add the "Create an Agent Integration" and "Create an API Integration" docs to the "Further Reading" section?